src/main/java/time/MonthView.java

```diff
@@ -93,15 +99,50 @@ private void printTasksForWeek(LocalDate weekStart, int maxTasks, TaskManager ta
         }
     }
 
+
     private void printTaskForDay(List<Task> dayTasks, int taskIndex) {
         if (taskIndex < dayTasks.size()) {
             Task task = dayTasks.get(taskIndex);
-            System.out.printf(uiRenderer.TASK_DISPLAY_FORMAT, task.getName());
+            printTaskIcon(task);
         } else {
             System.out.print(uiRenderer.EMPTY_TASK_DISPLAY_FORMAT);
         }
     }
 
+    private void printTaskIcon(Task task) {
+        String taskIcon = getTaskIcon(task);
+        System.out.printf(uiRenderer.ICON_DISPLAY_FORMAT, taskIcon);
+    }
+
+    private String getTaskIcon(Task task) {
+        String taskIcon;
+        switch (task.getTaskType()) {
+        case "E":
+            taskIcon = getColoredTaskIcon(task, EVENT_ICON_COLOR);
+            break;
+        case "D":
+            taskIcon = getColoredTaskIcon(task, DEADLINE_ICON_COLOR);
+            break;
+        case "T":
+            taskIcon = getColoredTaskIcon(task, TODO_ICON_COLOR);
+            break;
```

Comment:
Very interesting idea to use color to represent taskTypes! We might want to look into testing on different platforms to see how it renders.